solver/Cargo.toml

```diff
@@ -3,17 +3,19 @@ name = "aries"
 version = "0.0.6"
 authors = ["Arthur Bit-Monnot <abitmonnot@laas.fr>"]
 edition = "2021"
-
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
+default = ["i32"]
 
 # If enabled, will instruct the the solver to count cpu cycles at various point of its execution.
 # The implementation relies to time-stamp counter and intrinsic for the x86_64 platform.
 # If the target platform is not supported, activating this feature will have no effects.
 cpu_cycles = []
 
-
+i32 = []
```

Comment:
Short comment about what these feature are used for would be helpful.

Reply:
Fixed with ffe254001767bfcd5842e2db6044ab9fb1d54471 "simplify integer features"

This commit also simplifies how integer features are handled in general. Now instead of having a cascade of features where "i128" overrides "i64" which overrides "i32", we just have two features: "i128" and "i64".

If neither is enabled then we just use the normal `i32` for IntCst and `i64` for LongCst. If both are enabled then it is a compile time error. Otherwise, they work the same as they did in the past.